app/settings.py

```diff
@@ -379,3 +379,5 @@
 ]
 
 SILENCED_SYSTEM_CHECKS = ['fields.W342',]
+
+FIRST_YEAR = 1857 # Year of patent filing of the 1st sound recorder by Édouard-Léon Scott de Martinville
```

Comment:
il faut être + explicite, par ex : `FIRST_YEAR_PUBLISHED`

Reply:
Bravo for the historical comment !

Reply:
Il y a surtout (si ma mémoire est bonne), la période d'enregistrement :
- recorded_from_year
- recorded_to_year
FIRST_YEAR_PUBLISHED = dates de publication (pour un disque)

Reply:
@josimonnot oui il y a en effet 2 dates butoir. Ms comme ce sont les mm et qu'on veut éviter des doublons de paramètre, on en définit qu'une seule.